.pre-commit-config.yaml

```diff
@@ -56,3 +56,12 @@ repos:
     require_serial: true
     stages: [push]
     types: [python]
+
+  - id: bump-version
+    name: bump-version
+    description: "Bumped Version: bump the version when a new commit come in"
+    language: system
+    entry: make version
+    require_serial: true
+    stages: [push]
+    types: [python]
```

Comment:
`make version` command will bump the version in `VERSION` file automatically when you commit it.  This feature will be great for python-wechaty.